src/main/java/uk/gov/hmcts/darts/arm/service/impl/AbstractArmBatchProcessResponseFiles.java

```diff
@@ -329,6 +336,10 @@ && nonNull(armResponseBatchData.getUploadFileFilenameProcessor())) {
         );
     }
 
+    protected void markEodAsResposneProcessingFailed(ExternalObjectDirectoryEntity externalObjectDirectory, UserAccountEntity userAccount) {
```

Comment:
You have a typo in your method name here response

Reply:
Good spot updated